scripts/install

```diff
@@ -197,20 +190,19 @@ ynh_add_systemd_config
 chown -R root:root "$final_path"
 
 # Set permissions to app files
-chown -R "$app":"$app" "$final_path/data/"
+chown -R $app:$app "$final_path/data/"
 chmod -R 750 "$final_path/data/"
 
 # Set permissions to config file
-chown "$app" "$final_path/config.ini"
+chown $app "$final_path/config.ini"
 chmod 600 "$final_path/config.ini"
 
 #=================================================
 # RELOAD NGINX
 #=================================================
+ynh_script_progression --message="Reloading NGINX web server..."
 
-ynh_script_progression --message="Reloading nginx"
-service "$app" start
-service nginx reload
+ynh_systemd_action --service_name=nginx --action=reload
```

Comment:
Don't we need to start the service like before (`service "$app" start`)?

Reply:
Like in the restoration script something like: `ynh_systemd_action --service_name=$app --action="start"`?

Reply:
Yes indeed!

Reply:
yes! I forgot 😅